lib/CCreatureSet.h

```diff
@@ -95,6 +95,10 @@ class DLL_LINKAGE CStackInstance : public CBonusSystemNode, public CStackBasicDe
 	ArtBearer::ArtBearer bearerType() const override; //from CArtifactSet
 	virtual std::string nodeName() const override; //from CBonusSystemnode
 	void deserializationFix();
+
+private:
+	void copyOppositeBonusesFromCreature(const CCreature * creature);
+	STRONG_INLINE void removeOppositeBonuses();

```

Comment:
strong inline should only be used in extreme cases where you are sure compiler won't optimize better than you and there is real performance impact so maybe it's better to remove it if you are unsure about using it